docs/requirements.md

```diff
@@ -2,88 +2,78 @@
 
-Given the previous values, the total number of resources required should be the "per-analysis" amount times the number of concurrent analysis.
+**NOTE:**
+For microk8s we added 1.5 CPU and 1.5 GB extra in the "Platform" meant to be used by microk8s itself.
 
-### Example
+| Installation type              | ~ Total vCPUs | ~ Total Memory | Replicas per component | Max. commits analyzed concurrently | Platform vCPUs | Platform Memory | Analysis Workers vCPUs | Analysis Workers Memory |
+| ------------------------------ | ------------- | -------------- | ---------------------- | ---------------------------------- | -------------- | --------------- | ---------------------- | ----------------------- |
+| MicroK8s Minimum               | 16            | 32 GB          | 1                      | 2                                  | 5.5            | 9.5 GB          | 10                     | 20 GB                   |
+| MicroK8s Recommended (default) | 20+           | 32+ GB         | 1-2                    | 2                                  | 11+            | 20+ GB          | 10                     | 20 GB                   |
 
-_Maximum of 6 concurrent analysis_
+### Storage
 
-```yaml
-worker-manager:
-  config:
-    workers:
-      dedicatedMax: 3
-    workerResources:
-      limits:
-        cpu: 1
-        memory: "2Gi"
-    pluginResources:
-      requests:
-        cpu: 0.5
-        memory: 1000000000 # 1GB
-```
+The storage requirements depend mainly on the number of repositories you will be analyzing.
+Use the following table as a guideline to determine Codacy's storage requirements.
 
-In this example the minimum number of resources required would be:
+| Component | Bundled in the chart         | Minimum Recommended |
+| --------- | ---------------------------- | ------------------- |
+| NFS       | Yes                          | 200 GB              |
+| RabbitMQ  | Yes                          | 8 GB                |
+| Minio     | Yes                          | 20 GB               |
+| Postgres  | No (external DB recommended) | 500 GB+             |
 
-```text
-CPU: 6 * (1 + (0.5 * 4)) = 18 CPU
-Memory: 6 * (2 + (1 * 4)) = 36 GB
-```
+For a custom recommendation, please contact us at [support@codacy.com](mailto://support@codacy.com).
 
-**Total:**
+## External PostgreSQL instance
 
-```text
-CPU: 7 CPU + 18 CPU = 25 CPU
-Memory: 40 GB + 36 GB = 76 GB
-```
+Codacy requires a working PostgreSQL instance to persist data.
 
-## PostgreSQL server setup
+Google itself [doesn't recommend running database servers on your cluster](https://cloud.google.com/blog/products/databases/to-run-or-not-to-run-a-database-on-kubernetes-what-to-consider). As such, consider using a managed a solution like AWS RDS or Google Cloud SQL, or running the PostgreSQL server inside a dedicated virtual machine.
```

Comment:
@gjsduarte had some comments on the other PR 
https://github.com/codacy/chart/pull/316#discussion_r399462784